mw-webapp/src/utils/DateValue.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Date in ISO format yyyy-mm-dd
```

Comment:
why do you need a class for just one specific formatting helper?

Reply:
In the last rewview you wrote that instead of helpers that returns date better to use class with static methods. So I created class. Right now I use only current date that why it include one method.